lib/tasks/dev.rake

```diff
@@ -0,0 +1,33 @@
+namespace :dev do
+  require 'factory_bot_rails'
+  require 'faker'
+
+  desc 'Generate 100 developer record'
+  task :generate_data, [:developers_num] => :environment do |_, args|
+    set_args(args)
+
+    %w(ruby php java react).each { |name|
+      next name if ProgrammingLanguage.exists?(name: name)
+      FactoryBot.create(:programming_language, name: name)
```

Comment:
You hit the database twice with `exists?` and `create` for non existing record, which is inefficient.
You can use `find_or_create_by`.